Source/Extensions/Blazorise.DataGrid/Contexts/GroupContext.cs

```diff
@@ -0,0 +1,43 @@
+﻿#region Using directives
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Components;
+#endregion
+
+namespace Blazorise.DataGrid;
+
+/// <summary>
+/// Context for a group.
+/// </summary>
+/// <typeparam name="TItem"></typeparam>
+public class GroupContext<TItem>
```

Comment:
Name it DataGridGroupContext

Reply:
Uhm I believe that for Datagrid and since its on its own namespace we didn't do Datagrid prefix for the context? See other existing contexts... Filter, edit etc?

Reply:
Hmm you're right. OK leave it without prefix.